frontend/src/component/pages/Home.tsx

```diff
@@ -253,6 +270,12 @@ export class Home extends Component<Props, State> {
           </Section>
         </div>
         <Footer
+          uiFactory={this.props.uiFactory}
+          changeLog={this.state.changeLog}
+          newUpdateReleased={this.state.newUpdateReleased}
+          updateLastSeenChangeLog={
```

Comment:
updateLastSeenChangeLog will update cookie with key "whatsNewSettings.lastSeenDate" to number of milliseconds elapsed since epoch time.